.github/workflows/ci.yaml

```diff
@@ -13,12 +13,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
```

Comment:
I'm a fan of just removing this and inlining the versions below, since we don't actually maintain the boilerplate (or any application) for multiple elixir/erlang versions

Reply:
Oh right, I don’t know why we’re using a matrix strategy here 🤔

Reply:
Will do!

Reply:
@JoeyBG However, the `matrix` is helpful the cache key interpolation. Is there any "variables" concepts available in _GitHub Actions_?

Reply:
Yes, I think you’ll have to use [environment variables](https://docs.github.com/en/actions/learn-github-actions/environment-variables).

Reply:
> Yes, I think you’ll have to use [environment variables](https://docs.github.com/en/actions/learn-github-actions/environment-variables).

I also saw this in the documentation, but was not sure if it made sense to uses `ENV` for this purpose 🤔